pose_conversions.py

```diff
@@ -0,0 +1,262 @@
+    Args:
+        transformation_matrix: a numpy array (4x4)
+
+    Returns:
+        AxisAngle
+
+    """
+    rotation_matrix = transformation_matrix[:3, :3]
+    rotation = R.from_matrix(rotation_matrix)
+    return AxisAngle(rotation.as_rotvec())
+
+
+def zivid_to_rotation_vector(transformation_matrix):
+    """Convert from Transformation Matrix --> Rotation Vector.
+
+    Args:
+        transformation_matrix: a numpy array (4x4)
+
+    Returns:
+        Rotation Vector
+
+    """
+    axis_angle = zivid_to_axis_angle(transformation_matrix)
+    return axis_angle.as_rotvec()
+
+
+def zivid_to_quaternion(transformation_matrix):
+    """Convert from Transformation Matrix --> Quaternion.
+
+    Args:
+        transformation_matrix: a numpy array (4x4)
+
+    Returns:
+        Quaternion
+
+    """
+    rotation_matrix = transformation_matrix[:3, :3]
+    rotation = R.from_matrix(rotation_matrix)
+    return rotation.as_quat()
+
+
+def axis_angle_to_rotation_matrix(axis_angle: AxisAngle):
+    """Convert from AxisAngle --> Rotation Matrix.
+
+    Args:
+        axis_angle: an AxisAngle object with axis and angle
+
+    Returns:
+        Rotation Matrix (3x3 numpy array)
+
+    """
+    return R.from_quat(axis_angle.as_quaternion()).as_matrix()
+
+
+def rotation_vector_to_rotation_matrix(rotvec):
+    """Convert from Rotation Vector --> Rotation Matrix.
+
+    Args:
+        rotvec: a 3x1 numpy array
+
+    Returns:
+        Rotation Matrix (3x3 numpy array)
+
+    """
+    return R.from_rotvec(rotvec).as_matrix()
+
+
+def quaternion_to_rotation_matrix(quaternion):
+    """Convert from Quaternion --> Rotation Matrix.
+
+    Args:
+        quaternion: a 4x1 numpy array
+
+    Returns:
+        Rotation Matrix (3x3 numpy array)
+
+    """
+    return R.from_quat(quaternion).as_matrix()
+
+
+def save_transformation_matrix_to_yaml(transformation_matrix, path: Path):
+    """Save Transformation Matrix to YAML. Uses OpenCV to maintain yaml format.
+
+    Args:
+        transformation_matrix: 4x4 Transformation Matrix
+        path: path to save the YAML output
+
+    """
+    file_storage_out = cv2.FileStorage(path, cv2.FILE_STORAGE_WRITE)
+    file_storage_out.write("TransformationMatrixFromQuaternion", transformation_matrix)
+    file_storage_out.release()
+
+
+def get_transformation_matrix_from_yaml(path: Path):
+    """Get Transformation Matrix from YAML. Uses OpenCV to maintain yaml format.
+
+    Args:
+        path: path to the YAML file
+
+    Returns:
+        4x4 Transformation Matrix
+
+    """
+    file_storage_in = cv2.FileStorage(path, cv2.FILE_STORAGE_READ)
+    transformation_matrix = file_storage_in.getNode("PoseState").mat()
+    file_storage_in.release()
+    return transformation_matrix
+
+
+def print_header(txt: str):
+    """Print decorated header.
+
+    Args:
+        txt: Text to be printed in header
+
+    """
+    print()
+    print(f"{'*' * 70}")
+    print(f"* {txt} {' ' * (70 - len(txt) - 4)}*")
+    print(f"{'*' * 70}")
```

Comment:
Not sure what the purpose of the this function is. Can you show an example print here in this PR?

Reply:
```
print_header("Convert from Zivid to Robot")
```

yields:

```
**********************************************************************
* Convert from Zivid to Robot                                        *
**********************************************************************
```